### PR TITLE
Created a SettingsPage checklist

### PR DESCRIPTION
**Pull request by KirilenkoI**

### CODE REVIEW

File: docs/qa/CheckLists/SettingsPage.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+|    | 26 Settings                                                                                                                                                |             | Design page 26                                                                                                                          |
+| 1  | Check all UI texts/elements on design page 26                                                                                                              |             |                                                                                                                                         |
+| 2  | Pressing "Sound" radio button enables/disables sound of the app                                                                                            |             | Is it correct? Or does it disable notifications?                                                                                        |
```
Comment (vamelchenia) [inline docs/qa/CheckLists/SettingsPage.md:5]: Yes, if we include this feature.

File: docs/qa/CheckLists/SettingsPage.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+| 3  | Pressing "Show welcome page" radio button enables/disables appearance of "welcome" page (design page 1)                                                    |             |                                                                                                                                         |
```
Comment (vamelchenia) [inline docs/qa/CheckLists/SettingsPage.md:6]: "Welcome" instead of "welcome"

File: docs/qa/CheckLists/SettingsPage.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+| 5  | All 4 options of "Clear Recycle bin every" work as intended                                                                                                |             |                                                                                                                                         |
+| 6  | Pressing "Proceed to Recycle bin" button leads to recycle bin page (design page 28)                                                                        |             |                                                                                                                                         |
+| 7  | Pressing "Help" button leads to help page (design page 27)                                                                                                 |             |                                                                                                                                         |
```
Comment (vamelchenia) [inline docs/qa/CheckLists/SettingsPage.md:10]: Quotes and capitalisation missing

File: docs/qa/CheckLists/SettingsPage.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+| 8  | Pressing "Licenses" button leads to somewhere                                                                                                              |             |                                                                                                                                         |
```
Comment (vamelchenia) [inline docs/qa/CheckLists/SettingsPage.md:11]: to Licenses info

File: docs/qa/CheckLists/SettingsPage.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+|    | 27 Help                                                                                                                                                    |             |                                                                                                                                         |
+| 9  | Check all UI texts/elements on (design page 27)                                                                                                            |             | Design page 27                                                                                                                          |
+| 10 | When page opens up all questions are hidden behind headers                                                                                                 |             | Will state of open/close saved between sessions? Example, user opens up F.A.Q and there already opened questions that he open up before |
```
Comment (vamelchenia) [inline docs/qa/CheckLists/SettingsPage.md:14]: open**s**

**Review by vamelchenia — CHANGES_REQUESTED** [review]
Left comments

File: docs/qa/CheckLists/SettingsPage.md (hunk 1 above)
Comment (KirilenkoI) [reply docs/qa/CheckLists/SettingsPage.md:5]: Change comment to ("subject to change")

File: docs/qa/CheckLists/SettingsPage.md (hunk 2 above)
Comment (KirilenkoI) [reply docs/qa/CheckLists/SettingsPage.md:6]: Changed welcome to Welcome

File: docs/qa/CheckLists/SettingsPage.md (hunk 4 above)
Comment (KirilenkoI) [reply docs/qa/CheckLists/SettingsPage.md:11]: Changed to 'Pressing "Licenses" button leads to to Licenses info'

File: docs/qa/CheckLists/SettingsPage.md (hunk 5 above)
Comment (KirilenkoI) [reply docs/qa/CheckLists/SettingsPage.md:14]: Changed to "When page is opened all questions are hidden behind headers"

File: docs/qa/CheckLists/SettingsPage.md (hunk 3 above)
Comment (KirilenkoI) [reply docs/qa/CheckLists/SettingsPage.md:10]: Added quotes to 1) recycle bin page  2) help page  3) license info page

**Review by vamelchenia — APPROVED** [review]

**Pull request merged**